doc/release-notes/rl-2511.section.md

```diff
@@ -33,4 +33,6 @@
 
 ### Additions and Improvements {#sec-nixpkgs-release-25.11-lib-additions-improvements}
 
-- Create the first release note entry in this section!
+- Neovim Wrapper: Added support for the `vim.o.exrc` option, as well as the `VIMINIT` environment variable, and sourcing of `sysinit.vim`.
```

Comment:
I'd need to check the rl2505 entries, but I think release notes usually refer to the package attrname:


```suggestion
- `neovim`: Added support for the `vim.o.exrc` option, the `VIMINIT` environment variable, and sourcing of `sysinit.vim`.
```

Or they might use `pkgs.neovim` (I can't remember)